cadquery/cq.py

```diff
@@ -2855,20 +2853,25 @@ def _cutFromBase(self, obj: Shape) -> "Workplane":
 
         return self.newObject([r])
 
-    def combine(self, clean: bool = True) -> "Workplane":
+    def combine(
+        self, clean: bool = True, glue: bool = False, tol: Optional[float] = None
+    ) -> "Workplane":
         """
         Attempts to combine all of the items on the stack into a single item.
         WARNING: all of the items must be of the same type!
 
         :param boolean clean: call :py:meth:`clean` afterwards to have a clean shape
+        :param boolean glue: use a faster gluing mode for non-overlapping shapes (default False)
+        :param float tol: tolerance value for fuzzy bool operation mode (default None)
         :raises: ValueError if there are no items on the stack, or if they cannot be combined
         :return: a CQ object with the resulting object selected
         """
+
         items: List[Shape] = [o for o in self.objects if isinstance(o, Shape)]
         s = items.pop(0)
 
         if items:
-            s = s.fuse(*items)
+            s = s.fuse(*items, glue=glue, tol=tol)
 
```

Comment:
Having the tol parameter here won't cause problems when glue is false will it?

Reply:
They are orthogonal, so no issue.